xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1387,6 +1387,110 @@ class FlashAttentionPagedAttention : public MultiHeadedAttentionTest {
   }
 };
 
+class FlashAttentionFlexAttention : public MultiHeadedAttentionTest {
+ protected:
+  const std::string
+  GetModuleFlash_Attention_Flex_Attention_Soft_Capping_BF16() {
+    const std::string hlo_text = R"(
```

Comment:
Can you make this variable a `static constexpr absl::string_view kHloText = ...` and change the return type to `absl::string_view`?